README.rst

```diff
@@ -1,203 +1,128 @@
 =========
-Eox-hooks
+EOX Hooks
 =========
 
-.. |build-status| image:: https://circleci.com/gh/eduNEXT/eox-hooks.svg?style=svg
+|Maintainance Badge| |Test Badge| |PyPI Badge|
 
-Eox-hooks (A.K.A. Edunext Open extensions) is an `openedx plugin`_, for the `edx-platform`_ that allows extending
-edx-platform through Django configurations and Open edX Events.
+.. |Maintainance Badge| image:: https://img.shields.io/badge/Status-Maintained-brightgreen
+   :alt: Maintainance Status
+.. |Test Badge| image:: https://img.shields.io/github/actions/workflow/status/edunext/eox-hooks/.github%2Fworkflows%2Ftests.yml?label=Test
+   :alt: GitHub Actions Workflow Test Status
+.. |PyPI Badge| image:: https://img.shields.io/pypi/v/eox-hooks?label=PyPI
+   :alt: PyPI - Version
 
-Compatibility Notes
---------------------
+This plugin extends and customizes the `edx-platform`_ by leveraging Open edX and Django events to trigger specific actions. Key features include:
 
-+-------------------+----------------+
-| Open edX Release  |  Version       |
-+===================+================+
-|       Juniper     |   <= 1.0       |
-+-------------------+----------------+
-|        Koa        |    > 1.0       |
-+-------------------+----------------+
-|       Lilac       |    > 1.0 < 2.0 |
-+-------------------+----------------+
-|       Maple       |   >= 2.0 < 3.0 |
-+-------------------+----------------+
-|       Nutmeg      |   >= 3.0       |
-+-------------------+----------------+
-|       Olive       |   >= 4.0       |
-+-------------------+----------------+
-|       Palm        |   >= 5.0       |
-+-------------------+----------------+
-|       Quince      |   >= 6.0       |
-+-------------------+----------------+
+- **Post to Webhook URL:** Sends data to a specified URL based on the eox-hooks settings.
+- **Get Extended Certificate Data:** Adds comprehensive certificate information to the webhook request.
+- **Trigger Enrollments Creation:** Initiates an asynchronous task to enroll users in a list of courses.
+- **Trigger Grades Assignment:** Propagates grades to a specified course component.
+- **Create Enrollments for Program:** Enroll users in a predefined list of courses constituting a Course Program.
 
-The following changes to the plugin settings are necessary. If the release you are looking for is
```

Comment:
Can we maintain an explanation of the settings as suggested here https://github.com/eduNEXT/eox-theming/pull/55/files#r1687159312

Reply:
I'll add this:

> The plugin is configured for the latest release (Quince). The following changes in the plugin settings should be applied to be used for previous releases.
> Those settings can be changed in `eox_hooks/settings/common.py`` or, for example, in the instance configurations.
> 
> 
> 🚨 If the release you are looking for is not listed, please note:
> 
> - If the Open edX release is compatible with the current eox-hooks version (see `Compatibility Notes <https://github.com/eduNEXT/eox-hooks?tab=readme-ov-file#compatibility-notes>`_), the default configuration is sufficient.
> - If incompatible, you can refer to the README from the relevant version tag for configuration details (e.g., `v5.1.0 README <https://github.com/eduNEXT/eox-hooks/blob/v5.1.0/README.rst>`_).

Is that okay?

Reply:
yes